cmd/vic-init/ops.go

```diff
@@ -69,6 +70,9 @@ func (t *operations) HandleSessionExit(config *tether.ExecutorConfig, session *t
 			return
 		}
 
+		// incredibly basic throttle
+		time.Sleep(3 * time.Second)
```

Comment:
could do with testing in a situation that causes the tight restart loop.
Happy to start this much short if we have some sort of back-off - worst case here we're DoS'ing the infrastructure.